client/src/components/ElementEditor/tests/Header-test.js

```diff
@@ -170,7 +170,7 @@ describe('Header', () => {
         />
       );
 
-      expect(wrapper.text()).not.toContain('ElementActionsComponent');
+      expect(wrapper.text()).toContain('ElementActionsComponent');
```

Comment:
This is shallow rendered, and it won't go all the way to rendering the actual buttons, so I can't make assertions beyond checking that its rendering the action list or not.